### PR TITLE
Task15

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

**Comment by MrOlegKravchenko** [comment]
Better than a cup of coffee in the morning :)
image

**Comment by MrOlegKravchenko** [comment]
Please describe your commit messages as clear as you can, you should know what you did without watching the commit!

**Comment by yana-may25** [comment]
sorry. I know it's wrong
but very often I press this commit button and sth goes wrong
like, there's suddenly (u know, magic), a folder that should be in another branch 
or some changes still stayed
idk, I always mess that up :(

**Comment by yana-may25** [comment]
and it asks me to commit every time before switching to another branch, which pisses me off a bit

File: src/task15/task15.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class ArraySeq {
```
Comment (MrOlegKravchenko) [inline src/task15/task15.js]: Remove classes

Comment (yana-may25) [reply src/task15/task15.js]: why do u have it in for classes?

**Pull request merged**